packages/svelte/tests/runtime-runes/samples/side-effect-derived-set/_config.js

```diff
@@ -3,7 +3,6 @@ import { test } from '../../test';
 
 export default test({
 	compileOptions: {
-		dev: true,
```

Comment:
?

Reply:
Oh, wait, nope, I see what you're saying. Just an oops. Added it back